experiments/dataSources/worldBank/readme.md

```diff
@@ -0,0 +1,102 @@
+# Analyse World Bank statictical data in Skuid
+The World Bank provides free and open access to global development data.  Information like GDP, Population, Economic growth and Development measures are available for your Skuid pages. This is one of the easiest API to interact with. 
```

Comment:
```suggestion
The World Bank provides free and open access to global development data.  Information like GDP, Population, Economic growth and Development measures are available for your Skuid pages. This is one of the easiest APIs to interact with. 
```

Reply:
Copy nitpick: you say this is the easiest API here, and then you say it again in the next paragraph while also explaining why.

I'd say you could just cut this sentence here and leave the one with explanation below.